libs/sdk-core/src/breez_services.rs

```diff
@@ -1896,26 +1903,30 @@ impl Receiver for PaymentReceiver {
     }
 }
 
-/// Convenience method to look up LSP info based on current LSP ID
-async fn get_lsp(persister: Arc<SqliteStorage>, lsp: Arc<dyn LspAPI>) -> Result<LspInformation> {
+/// Convenience method to look up LSP info based on current LSP ID and
+/// fallback to the first LSP in the listed LSPs if the current LSP is not available.
```

Comment:
```suggestion
/// Convenience method to look up LSP info based on current LSP ID.
/// If none is selected, it selects and persists the first LSP in the listed LSPs as fallback.
```

Reply:
I reverted the comment, `get_lsp` behaves as before